src/toolkit/matl_buy_policy.cc

```diff
@@ -43,18 +45,44 @@ void MatlBuyPolicy::set_manager(Agent* m) {
   }
 }
 
+void MatlBuyPolicy::set_total_inv_tracker(TotalInvTracker* t) {
+  if (t == NULL){
+    std::vector<ResBuf<Material>*> bufs = {buf_};
+    buf_tracker_->Init(bufs, buf_->capacity());
+  }
+  else {
+    buf_tracker_ = t;
+  }
+}
+
+void MatlBuyPolicy::set_inv_policy(std::string inv_policy, double fill, double req_at) {
+  set_req_at(req_at);
+  if ((inv_policy == "sS") || (inv_policy == "Ss")) {
+    set_fill_to(fill);
+  }
+  else if ((inv_policy == "RQ") || (inv_policy == "QR")) {
+    set_quantize(fill);
+    // maximum amount that an RQ policy could achieve is req_at + fill
+    set_fill_to(req_at + fill);
+  }
+  else {
+    throw ValueError("Invalid inventory policy");
+  }
+}
+
 void MatlBuyPolicy::set_fill_to(double x) {
-  if (x > 1)
```

Comment:
This seems like an important difference - can you explain why we no longer want to constrain this to [0,1]?

Reply:
Sure. `fill_to` and `req_at`, as they were originally imagined, were set as a fraction of the capacity. Through re-factoring the code and using them in my own demonstrations, I found it very annoying to work with these as fractions and felt that they would be better off as actual mass values